src/util/tests/angle-calculation.test.ts

```diff
@@ -0,0 +1,8 @@
+import { radiansToDegrees } from "../angle-calculation"
+
+test('Convert correctly radians to degrees', () => {
```

Comment:
Would floating point deviations cause this test to fail? Should we specify the number of decimals that we should check for?

Reply:
Yeah, better safe than sorry. I added `toBeCloseTo`  to compare floating-point https://jestjs.io/docs/en/using-matchers#numbers